demomultistoreform/demomultistoreform.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+if (file_exists(__DIR__.'/vendor/autoload.php')) {
+    require_once __DIR__.'/vendor/autoload.php';
+}
+
+use PrestaShop\PrestaShop\Adapter\SymfonyContainer;
+use PrestaShop\Module\DemoMultistoreForm\Database\ContentBlockInstaller;
+use PrestaShop\Module\DemoMultistoreForm\Database\ContentBlockGenerator;
+
+class DemoMultistoreForm extends Module
+{
+    public $multistoreCompatibility = self::MULTISTORE_COMPATIBILITY_YES;
+
+    public function __construct()
+    {
+        $this->name = 'demomultistoreform';
+        $this->author = 'PrestaShop';
+        $this->version = '1.0.0';
+        $this->need_instance = 0;
+
+        $this->bootstrap = true;
+        parent::__construct();
+
+        $this->displayName = $this->trans('Demo multistore configuration form', array(), 'Modules.DemoMultistoreForm.Admin');
+        $this->description = $this->trans(
```

Comment:
🆙

Reply:
I have no idea to be honest... I could just remove the usage of translation and use strings instead ? That's not really the subject of this module

Reply:
> I could just remove the usage of translation and use strings instead ? That's not really the subject of this module

I think it would be safer indeed 👍 

Not blocking